rules/opentitan.bzl

```diff
@@ -42,6 +42,11 @@ PER_DEVICE_DEPS = {
     "fpga_cw310": ["@//sw/device/lib/arch:fpga_cw310"],
 }
 
+# Default keys used to sign ROM_EXT and Owner images for testing.
```

Comment:
nit: I know we haven't been very consistent about this but going forward we can refer to the owner stage image shortly as BL0 in code. No need to change this line.